policy/modules/contrib/ldap.if

```diff
@@ -216,9 +216,6 @@ interface(`ldap_admin',`
 
 	admin_pattern($1, slapd_lock_t)
 
-	files_list_var_lib($1)
```

Comment:
I believe you need to keep this line, to allow accessing `/var/lib/openldap-data/...` (`slapd_db_t`). Please move it above the `admin_pattern()` that contains `slapd_db_t`,

Reply:
Thanks, done!